.github/workflows/rollup-config-diff.yml

```diff
@@ -0,0 +1,39 @@
+name: Rollup Config Diff
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Looks like this is the default set so this is ok, but what is synchronize? I don't see an explanation of it in the [github docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

Reply:
I believe it is when a new commit is pushed to a PR.